File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/swift/F16.P2.swift
func mapTotal() -> Int32 {
// BEGIN PROBE F16.P2
let mp: [String: Int32] = ["a": 1]
var total: Int32 = 0
for e in mp { total += e.value }
let miss = mp["b"] ?? 0
return total + miss
// END PROBE F16.P2
}

print(mapTotal())

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/go/F16.P2.go
package main

import "fmt"

func mapTotal() int32 {
	// BEGIN PROBE F16.P2
	mp := map[string]int32{"a": 1}
	var total int32
	for _, v := range mp {
		total += v
	}
	miss := mp["b"]
	return total + miss
	// END PROBE F16.P2
}

func main() {
	fmt.Println(mapTotal())
}
